networkx/algorithms/bipartite/tests/test_extendability.py

```diff
@@ -29,6 +29,14 @@ def test_no_perfect_matching_raises():
         nx.bipartite.maximal_extendability(G)
 
 
+def test_not_connected_raises():
```

Comment:
`test_not_strongly_connected` might be more appropriate.

Reply:
Oh yes! Thankyou for pointing this out.